projects/1.18/assets/andycats-flowers/andycats_flowers/lang/zh_cn.json

```
@@ -0,0 +1,141 @@
+{
+  "block.andycats_flowers.cyan_cosmos": "青色大波斯菊",
+  "block.andycats_flowers.purple_gladiolus": "紫色剑兰",
+  "block.andycats_flowers.tall_cyan_gladiolus": "高青色剑兰",
+  "block.andycats_flowers.pink_mallow": "粉红色锦葵",
+  "block.andycats_flowers.tall_white_daffodil": "高白色水仙",
+  "block.andycats_flowers.cyan_echinacea": "青色锥菊",
+  "block.andycats_flowers.tall_red_cosmos": "高红色大波斯菊",
+  "block.andycats_flowers.tall_purple_iris": "高紫色鸢尾",
+  "block.andycats_flowers.tall_purple_echinacea": "高紫色锥菊",
+  "block.andycats_flowers.light_blue_commelina": "淡蓝色鸭跖草",
+  "block.andycats_flowers.purple_iris": "紫色鸢尾",
+  "block.andycats_flowers.white_echinacea": "白色锥菊",
+  "block.andycats_flowers.tall_maroon_mallow": "高褐红色锦葵",
+  "block.andycats_flowers.tall_white_gladiolus": "高白色剑兰",
+  "block.andycats_flowers.blue_cowslip": "蓝色樱草",
```

```suggestion
  "block.andycats_flowers.blue_cowslip": "蓝色报春花",
```
看了下维百，cowslip对应的物种不一，这里用一个排首位的[*Primula veris*](http://www.iplant.cn/info/%E9%BB%84%E8%8A%B1%E4%B9%9D%E8%BD%AE%E8%8D%89)，但大多和[樱草](http://www.iplant.cn/info/%E6%A8%B1%E8%8D%89)一样属于报春花属
仅是建议

---

可以